app/controllers/equipment_models_controller.rb

```diff
@@ -58,6 +58,17 @@ def show
     @max = @equipment_model.equipment_items.active.count
 
     @restricted = @equipment_model.model_restricted?(cart.reserver_id)
+
+    # For pending reservations table
```

Comment:
See [below](https://github.com/YaleSTC/reservations/pull/1206/files#r27473905) for an updated about this scope / variable.